Windowmove: Allow percentages for width and height

Collinux opened the pull request:
In reponse to [windowmove should accept percentages issue #27](https://github.com/jordansissel/xdotool/issues/27), added support for width and height percentages in windowmove.

jordansissel commented:
Some weird whitespace changes in this PR, but otherwise the code LGTM.

Can you make a note in `xdotool.pod` for the windowmove command to document the new percentage syntax?

Collinux commented:
@jordansissel I don't know why the diff keeps showing I added and removed lines back... I agree it's weird but if you look at the final code I tried my best to follow your format.

jordansissel commented:
No worries on the whitespace changes :)


On Tue, Jan 5, 2016 at 1:22 PM, Collin Guarino <notifications@github.com>
wrote:

> @jordansissel <https://github.com/jordansissel> I don't know why the diff
> keeps showing I added and removed lines back... I agree it's weird but if
> you look at the final code I tried my best to follow your format.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jordansissel/xdotool/pull/92#issuecomment-169137723>.
>
